src/rtos/rtos.cpp

```diff
@@ -14,9 +14,12 @@
 
 #include "pros/rtos.hpp"
 
+#include <atomic>
 #include <system_error>
 
 #include "kapi.h"
+#include "pros/rtos.h"
```

Comment:
Replace this with `#include rtos/task.h`

Reply:
Addressed in https://github.com/purduesigbots/pros/pull/677/commits/26f77c86587f6ac3e6fb0991f71622199ac848f1.